html-formatter/javascript/check.js

```diff
@@ -0,0 +1,22 @@
+const fs = require('fs')
+const path = process.argv[2]
+const html = fs.readFileSync(path, 'utf-8')
+const puppeteer = require('puppeteer');
+
+(async () => {
+  const browser = await puppeteer.launch();
+  const page = await browser.newPage();
+  await page.setContent(html)
+  await page.pdf({path: 'hn.pdf', format: 'A4'});
```

Comment:
What purpose does the PDF generation serve?

Reply:
re: pdf. Copy paste error - removed